packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1252,83 +1256,91 @@ Link: <https://${yourOktaDomain}/api/v1/users?after=00ud4tVDDXYVKPXKVLCO&limit=2
 ]
 ```
 
-#### Find Users
-
-
-Finds users who match the specified query
-
-Use the `q` parameter for a simple lookup of users by name, for example when creating a people picker.
-The value of `q` is matched against `firstName`, `lastName`, or `email`.
-
+##### Searching arrays
 
-This operation:
+You can search properties that are arrays. If any element matches the search term, the entire array (object) is returned.
+For examples, see [Request example for array](#request-example-for-array) and [Response example for array](#response-example-for-array).
 
- * Doesn't support pagination.
- * Queries the most up-to-date data. For example, if you create a user or change an attribute and then issue a filter request, the change is reflected in the results.
- * Performs a startsWith match but this is an implementation detail and may change without notice. You don't need to specify `firstName`, `lastName`, or `email`.
+* We follow the [SCIM Protocol Specification](https://tools.ietf.org/html/rfc7644#section-3.4.2.2) for searching arrays.
+* Search for one value at a time when searching arrays. For example, you can't search for users where a string is equal to an attribute in two different arrays.
```

Comment:
I know you just moved this from somewhere else, but isn't this wrong?  Of course I can search for a string in two different arrays: `arrayAttr1 eq "foo" and arrayAttr2 eq "foo"`.  I _think_ what this is trying to say is you can't look for a single array that contains both "foo" and "bar".

Reply:
I'm not touching it.  I'm not sure it's wrong.

Reply:
Fine, where do I submit a new bug against this doc?

Reply:
I guess it's an ordinary OKTA-whatever Jira?  I do agree that it bears looking at, just not as part of this change.

Reply:
Filed https://oktainc.atlassian.net/browse/OKTA-488292